modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -474,6 +482,9 @@ sda.enkf.multisite <- function(settings,
         if(processvar == FALSE){an.method<-EnKF  }else{    an.method<-GEF.MultiSite   }
         
         #-analysis function
+
+        #source("~/pecan/modules/assim.sequential/R/Analysis_sda_multiSite.R")
+
```

Comment:
This looks like debug code -- I recommend removing

Reply:
Removing.